test/check-application

```diff
@@ -27,6 +27,11 @@ class TestFiles(testlib.MachineCase):
         self.login_and_go("/files")
         self.browser.wait_not_present(".pf-c-empty-state")
 
+    # TODO: move to testlib.py
+    def press_special_key(self, name):
+        args = {"type": "keyDown", "key": name}
+        self.browser.cdp.invoke("Input.dispatchKeyEvent", **args)
```

Comment:
This looks really nice! I'm not much of a fan of the "special" name, but we can debate that in the cockpit PR.

Reply:
Same, I cannot come up with a good name, so my idea was to land it here. Think some more and add it to Cockpit then `cockpit-lib-update`.

Reply:
How about `press_key()` and `type_text()`?

Reply:
Looked around a bit:

https://playwright.dev/docs/api/class-keyboard
https://pptr.dev/api/puppeteer.keyboard.down

keyboard_down('ArrowDown') seems like a good suggestion.

Reply:
Oh `press_key` / `type_text` sounds like selenium! WFM